NitroxClient/GameLogic/Spawning/DefaultEntitySpawner.cs

```diff
@@ -24,20 +24,43 @@ public Optional<GameObject> Spawn(Entity entity, Optional<GameObject> parent)
             }
 
             GameObject gameObject = Utils.SpawnFromPrefab(prefab, null);
+
             gameObject.transform.position = entity.Position;
-            gameObject.transform.localScale = entity.Scale;
+            gameObject.transform.localScale = entity.LocalScale;
+
+            if (parent.IsPresent() && parent.Get().name == "CellRoot(Clone)")
+            {
+                LargeWorldEntity ent = gameObject.GetComponent<LargeWorldEntity>();
+                ent.transform.SetParent(parent.Get().transform, true);
+                ent.OnAddToCell();
+
+                gameObject.transform.localPosition = entity.LocalPosition;
```

Comment:
minor: looks like these two lines are common so can be out of the if/else if/else